privacy-dashboard/src/main/java/com/github/privacydashboard/PrivacyDashboard.kt

```diff
@@ -174,16 +180,70 @@ object PrivacyDashboard {
      *
      * @param activity Activity to start activity
      */
+//    fun start(activity: Activity) {
```

Comment:
We dont want to push commented codes, unless it is very important, if so make sure you added comments(notes) why you pushed the commented codes